### DISCUSSION TITLE
Add more infectious diseases.

### DISCUSSION THREAD

**Discussion by kineticcrusher** [FakeFishGames/Barotrauma#8007] [Ideas and suggestions]
Currently, the only affliction that could be classified as 'contagious' is husk infection. While husks provide emergent difficulty and help to sew distrust among the crew, which makes the game a little more fun and interesting, there aren't really any other afflictions that can spread between crewmembers, especially none that can spread through bodily contact.

If humanity came to settle on an entirely new planet - one with native life, no less - there's no doubt that we'd get introduced to a lot of new diseases. As such, it would be good for both immersion and gameplay if some objects or creatures could infect the player with transmissible diseases. If a pathogen began spreading among the crew, it would add to the social deduction aspect of the game, as well as introducing need for things like quarantine rooms or biohazard suits.

An extra addition to this expansion could be contaminated items or surfaces, where sufficiently sick crewmates could spread disease just by touching an object which then gets touched by another crewmate.

**Comment by Likvidus** [comment]
Good idea, +1

**Comment by rav2n** [comment]
From myself I will add that it would be nice to make 1-2 representatives of the fauna poisonous in addition to Leucocytes. On our planet there are many marine inhabitants (mainly living in warm waters), a simple touch to which is guaranteed to end in death for a person if he is not provided with medical assistance in time. It would be strange if something similar did not appear in such an extreme place as Europe.
In addition, in the greenhouse conditions of the game, when there are no traitors or griefers, poisons and antidotes for them are guaranteed to gather dust somewhere in the far drawer of the medbay.

**Comment by Ek-Videogames** [comment]
### I would suggest caution as to not fall into the same design holes that plague the current implementation of poisons.
Right now poisons (and also paralysis) are just subtly-different death timers and a matching cure item respectively. 

There needs to be variety of meaningful mechanical impact or functional differences to justify their existence, otherwise it will just be another miscellaneous cure that you have to stock the submarine with and then not care about. The 'antidote' medicines already clog up almost half the medicine roster.

Otherwise agree with the idea.

**Reply by GoodGuyJimmy** [reply]
I never though of it that way but honestly, you're right, the poisons are pretty much the same and are not at all interesting, except for paralysis.

**Comment by ShrebTV** [comment]
A few infectious diseases would be really cool. 
They wouldnt even need special medical items to cure them, instead make them curable by a few doses of broad spectrum antibiotics given over a certain time.
Spinelings attacks, crawler bites, mudraptor bites, or just being in the water without a suit while having bite wounds, lacerations or bleeding could infect you. 
symptoms could be vomiting, blurred vision(fever), being slower(pain) or many other things like sneezing or coughing.

Overall I think general infectious diseases would be cool to play around especially in a bigger crew since people already sort of dont trust each other. 
coughing could also just be random without having an alien infection so people actually send others to the doc to get checked.

**Comment by GregRedfield** [comment]
I Would like to see reverse Husk infection treatment. I mean. Imagine your friend became husk and you need to cure him, but he is dangerous. And people begin to argue: No, we have to terminate him he is a danger to everyone. Otherwise, I have a husk infection. We will not get the antivirus in time, the end. Besides, when in Star Trek someone has some rare disease they go to a planet to gather some plant and make an antidote. How do we do that in one or two minutes of infection? 

Perhaps the husk infection could last longer, but sometimes you could lose control of yourself and become an AI for a few seconds. 

Thanks

**Comment by Prolbo** [comment]
Well, while Im not sure about contagious deseases, I'm absolutely sure that this game lacking simple Wound infection, which is strange. Each wound should have a chance to catch infection, gunshot wounds, bite wounds (!!!), tissue wounds, laserations and burns. First 3 should have a greater chanse to be infected at the moment when damage inflicted and much lesser over time. Burns, on the other hand, should have little to none chanse to catch infection while they are light and/or fresh and much greater chanse over time especially if they are really strong.
Also, I believe that you are all already figured what will help you to fight wound infection.

**Comment by LePiromano** [comment]
Instead of adding just another disease that could just be healed with another antidote, what if the infection wasn't actually healable but instead reduced the efficiency of healing drugs as it developed? As in, the more you advance in the campaign, the more the infection takes over the body (not like jovian radiation). This would push players to upgrade their medicine to something above bandages, saline and morphine.
